server/dash/dashboards/neighborhood_recent.py

```diff
@@ -22,8 +24,8 @@
     'address': "Address"
 }
 
-start_date = datetime.date.today() - datetime.timedelta(days=7)
-end_date = datetime.date.today() - datetime.timedelta(days=1)
+start_date = datetime.date.today() + datetime.timedelta(days=3)
```

Comment:
Why are the dates being changed here?

Also, please make constants for the `days` arguments here. E.g., START_DATE_DELTA_DAYS = 3.

Reply:
My bad - I was testing with different date ranges. Added the constants for the day arguments!